R/zzz.r

```diff
@@ -16,9 +16,21 @@
   })
 }
 
+# Assigning pathGrob in .onLoad ensures that packages that subclass GeomPolygon
+# do not install with error `possible error in 'pathGrob(munched$x, munched$y, ':
+# unused argument (pathId = munched$group)` despite the fact that this is correct
+# usage
+pathGrob <- NULL
```

Comment:
Just for my own understanding: Why do we need this assignment outside of `.onLoad()`?